### PR TITLE
Mailer working. testing on Heroku.

### PR DESCRIPTION
**Pull request by charkcl**

### CODE REVIEW

**Comment by dshcheung** [comment]
rebase with master

File: app/views/user_mailer/booking_confirmation.html.erb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<!-- render booking.show.html.erb? -->
+
+test-need to do
```
Comment (f3r) [inline app/views/user_mailer/booking_confirmation.html.erb]:
``` html
<p>Thank you for booking. Please find your booking details below.<p>
...
```

File: config/environments/production.rb [hunk 2]
```diff
@@ -77,3 +77,13 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 end
+
+ActionMailer::Base.smtp_settings = {
+  :port           => ENV['MAILGUN_SMTP_PORT'],
+  :address        => ENV['MAILGUN_SMTP_SERVER'],
+  :user_name      => ENV['MAILGUN_SMTP_LOGIN'],
+  :password       => ENV['MAILGUN_SMTP_PASSWORD'],
+  :domain         => 'yourapp.heroku.com',
```
Comment (f3r) [inline config/environments/production.rb]: pls update :domain too

**Comment by f3r** [comment]
let me know when you update the changes, thx

**Comment by charkcl** [comment]
Updated per F3r comments. Also did more scss on tours....

**Pull request merged**